test/fixtures/ember-template-lint-formatter-test/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "ember-template-lint-printer-test",
+  "version": "1.0.0",
+  "description": "test printer to show how to package printers and use them."
+}
```

Comment:
Should we require a custom keyword? E.g. `ember-template-lint-formatter`?

Reply:
We could. Are you suggesting we sniff the keywords?

Reply:
Lets not...